magento/pwa-studio
# Removes errant debugger statement in test file

## Description

There was an errant `debugger` statement left in a code file that would cause `yarn run test:debug` to pause execution when hit.

## Related Issue





Closes #1555.

## Verification Steps


You can run `yarn run test:debug` and attach a debugger and ensure that it doesn't break, or you can manually verify that I have deleted this `debugger` statement.

## Screenshots / Screen Captures (if appropriate)

## Checklist

* I have updated the documentation accordingly, if necessary.
* I have added tests to cover my changes, if necessary.
